src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -135,16 +136,14 @@
       (handle-prismatic-exception! e))))
 
 
-(defn parse-cli-args!
+(schema/defn parse-cli-args! :- common/CLIData
   "Parses the command-line arguments using `puppetlabs.kitchensink.core/cli!`.
   Hard-codes the command-line arguments expected by trapperkeeper to be:
       --debug
       --bootstrap-config <bootstrap file>
       --config <.ini file or directory>
       --plugins <plugins directory>"
-  [cli-args]
-  {:pre  [(sequential? cli-args)]
-   :post [(map? %)]}
+  [cli-args :- [(schema/maybe schema/Str)]]
```

Comment:
Seems like it would be better to have the schema reflect the real world data that we'd expect to see rather than loosening the schema to satisfy the needs of a unit test.

For that particular unit test, it seems like we're just wanting to validate that if no argument is provided to the right of "--bootstrap-config" that an appropriate error occurs.  Another way to do that might be to change the implementation of the `bootstrap-with-empty-config` function to put the `--config ...` option at the front of the argument list, like this:

```clj
(->> other-args
     (concat ["--config" empty-config])
     (vec)
```
... so that the test can just pass a value like ... 

```
(bootstrap-with-empty-config ["--bootstrap-config"])
```

... without requiring a bogus / non-real world `nil` argument to be appended to the end?